source/app/powertabeditor.cpp

```diff
@@ -1370,23 +1370,32 @@ void PowerTabEditor::editRepeatEnding()
 void PowerTabEditor::updateDynamic(Dynamic::VolumeLevel volume)
 {
     ScoreLocation &location = getLocation();
-    const Dynamic *dynamic = ScoreUtils::findByPosition(
+    const Dynamic *currentDynamic = ScoreUtils::findByPosition(
                 location.getStaff().getDynamics(), location.getPositionIndex());
+    Dynamic newDynamic(location.getPositionIndex(), volume);
 
-    if (dynamic)
-    {
-        myUndoManager->push(new EditDynamic(location, volume, dynamic),
-                            location.getSystemIndex());
-    }
-    else
+    if (currentDynamic)
     {
-        if (volume == dynamic->getVolume())
-            myUndoManager->push(new RemoveDynamic(location),
-                                location.getSystemIndex());
+        if (newDynamic.getVolume() == currentDynamic->getVolume())
+            {
```

Comment:
The indentation looks like it's off by 4 spaces here

Reply:
Fixed